tests/integration/simple_copy_test/test_simple_copy.py

```diff
@@ -266,7 +266,7 @@ def test__snowflake__incremental_overwrite(self):
         # Setting the incremental_strategy should make this succeed
         self.use_default_project({
             "models": {
-                "incremental_strategy": "delete+insert"
+                "incremental_strategy": "delete+insert",
```

Comment:
I know this is the nit of all nits, but can we remove this extraneous comma to restore the original configuration?
```suggestion
                "incremental_strategy": "delete+insert"
```